auth/saml/index.php

```diff
@@ -189,6 +189,8 @@
             $err['login'] = get_string("auth_saml_error_complete_user_data", "auth_saml", $username);
             saml_error($err['login'], '?logout', $pluginconfig->samllogfile);
         }
+
+        add_to_log(SITEID, "user", "login", "view.php?id=".$user->id."&course=".SITEID, $user->id, 0, $user->id);
```

Comment:
I believe you can implement `get_legacy_logdata` to add the log to both the new table and the legacy log